src/js/components/Typeahead.js

```diff
@@ -0,0 +1,83 @@
+import classNames from 'classnames';
+import InnerTypeahead from 'mesosphere-react-typeahead';
+/* eslint-disable no-unused-vars */
+import React, {PropTypes} from 'react';
+/* eslint-enable no-unused-vars */
+
+import FilterInputText from './FilterInputText';
+
+class Typeahead extends FilterInputText {
+  handleInputChange(input) {
+    this.props.handleFilterChange(input);
+  }
+
+  handleClearInput() {
```

Comment:
#nit Could we name these two handlers consistently? So you could just rename this to `handleInputClear`